great_ape_safe/ape_api/ebtc.py

```diff
@@ -784,6 +905,31 @@ def priceFeed_set_fallback_caller(
         if executed:
             assert self.price_feed.fallbackCaller() == address
 
+    def priceFeed_set_collateral_feed_source(
+        self, value, salt=EmptyBytes32, use_high_sec=False
```

Comment:
similar as other comment, something framing more readable of what `value` is will be sweet